indexing-engine/changelog.mdx

```diff
@@ -12,6 +12,17 @@ The dates on this page represent when features were first released for the cloud
 version of pganalyze. For pganalyze Enterprise Server releases, we indicate on each
 [release page](/docs/enterprise/releases) which release of the Indexing Engine is included.
 
+## 2022-08-09
+
+* Initial support for PostGIS
+  - Adds support for index analysis on queries that reference tables that have
+    PostGIS columns (or reference such tables through foreign key constraints) -
+    recommendations on actual PostGIS columns may still not work well, but we
```

Comment:
maybe "recommendations on actual PostGIS columns are not fully implemented yet and costs will not be accurate" 

(the main issue with PostGIS now is that we don't yet have enough table statistics + query constant values to somewhat accurately run the planner for expressions on such columns)